usecases/replica/finder.go

```diff
@@ -142,6 +144,46 @@ func (f *Finder) GetAll(ctx context.Context,
 	return result.Value, err
 }
 
+func (f *Finder) Search(ctx context.Context,
```

Comment:
I think `Search()` and related functions (`readSearch(), incoming_index. SearchObjects(), ...`) are not needed anymore.
We want to avoid the performance penalties related to delegating search to the finder and instead use `CheckConsistency()` which checks the consistency for the result set only

Reply:
Removed